Errors when trying the minimal setup

The minimal setup at the beginning of
https://zeit.co/blog/next

seems to generate cryptic errors:

```bash
ERROR  Failed to compile with 4 errors

These dependencies were not found in node_modules:

* ./pages/index.js?entry
* /Users/dmitrizaitsev/Dropbox/Sandbox/nextjs-test/node_modules/next/dist/pages/_error.js?entry
* /Users/dmitrizaitsev/Dropbox/Sandbox/nextjs-test/node_modules/next/dist/pages/_error-debug.js?entry
* /Users/dmitrizaitsev/Dropbox/Sandbox/nextjs-test/node_modules/next/dist/pages/_document.js?entry

Did you forget to run npm install --save for them?
> Ready on http://localhost:3000

```

`npm install` does not help :(

When I run `next build`. I saw an error like this:

> Module not found: Error: Can't resolve 'babel-loader'

And I change webpack config as below then resolve it.

```js
webpack: (config, { dev }) => {
    config.resolveLoader.modules.push(path.join(process.cwd(), 'node_modules'))
    return config
}
```

Maybe this is a difference between webpack@1 and webpack@2.

--- a/server/build/webpack.js
+++ b/server/build/webpack.js
@@ -47,7 +47,8 @@ export default async function createCompiler (dir, { dev = false, quiet = false 
     interpolateNames.set(path, `dist/pages/${p}`)
   }
 
-  const nodeModulesDir = join(__dirname, '..', '..', '..', 'node_modules')
+  const nextNodeModulesDir = join(__dirname, '..', '..', '..', 'node_modules')
+  const appNodeModulesDir = join(dir, 'node_modules')
   const minChunks = pages.filter((p) => p !== documentPage).length
 
   const plugins = [
@@ -184,8 +185,8 @@ export default async function createCompiler (dir, { dev = false, quiet = false 
     },
     resolve: {
       modules: [
-        nodeModulesDir,
-        join(dir, 'node_modules')
+        nextNodeModulesDir,
+        appNodeModulesDir
       ].concat(
         (process.env.NODE_PATH || '')
         .split(process.platform === 'win32' ? ';' : ':')
@@ -194,7 +195,8 @@ export default async function createCompiler (dir, { dev = false, quiet = false 
     },
     resolveLoader: {
       modules: [
-        nodeModulesDir,
+        nextNodeModulesDir,
+        appNodeModulesDir,
         join(__dirname, 'loaders')
       ]
     },

